pkg/db/client.go

```diff
@@ -238,6 +255,30 @@ func (t *TridentDB) ListCampaign() ([]Campaign, error) {
 	return campaigns, nil
 }
 
+// IsCampaignCancelled takes a campaign ID and returns true if the campaign status is CampaignStatusCancelled
+func (t *TridentDB) IsCampaignCancelled(campaignID uint) (bool, error) {
+	var matches []Campaign
+
+	var query = Query{
+		Filter: map[string]interface{}{"id": campaignID, "status": CampaignStatusCancelled},
+	}
+
+	err := t.db.
+		Where(query.Filter).
```

Comment:
Why do we create a Query if we only use the Filter value? Can we simply create a filter?